pkg/scaffold/v2/webhook_manager_patch.go

```diff
@@ -47,7 +47,7 @@ spec:
       containers:
       - name: manager
         ports:
-        - containerPort: 443
+        - containerPort: 8443
```

Comment:
Looking at the manager container log, I realized 8443 is already taken by auth proxy container.
See: https://github.com/kubernetes-sigs/kubebuilder/blob/76963b330e697c92399b1f0dace5f24c736b0768/testdata/project-v2/config/default/manager_auth_proxy_patch.yaml#L20

Reply:
Changing it to a different port should fix the failing test.

Reply:
It is true .. I could check it now .. tks .. it is one of my first contribution here .. so I am still learning how the things are working ..